src/vm/analysis/definition_sorter/mod.rs

```diff
@@ -94,11 +103,9 @@ impl <'a> DefinitionSorter {
                                 DefineFunctions::PrivateFunction | DefineFunctions::Constant |
                                 DefineFunctions::PublicFunction | DefineFunctions::PersistedVariable |
                                 DefineFunctions::ReadOnlyFunction => {
-                                    // Args: [(define-name-and-types), ...]: ignore 1st arg
-                                    if function_args.len() > 1 {
-                                        for expr in function_args[1..function_args.len()].into_iter() {
-                                            self.probe_for_dependencies(expr, tle_index)?;
-                                        }
+                                    // Args: [(define-name-and-types), ...]
```

Comment:
Why are we no longer ignoring the first argument?  Not sure which branch is right (or why this behavior change matters).

Reply:
Before the traits, traversing the 1st argument and looking for dependencies was useless, vs required now - we are looking for traits aliases.

Reply:
Gotcha :+1: